src/test/kotlin/org/rust/lang/core/resolve/RsResolveTest.kt

```diff
@@ -485,8 +485,8 @@ class RsResolveTest : RsResolveTestBase() {
     """)
 
     fun testForeignStatic() = checkByCode("""
-        extern "C" { static FOO: i32; }
-                            //X
+        extern "C" { static mut FOO: i32; }
+                                //X
```

Comment:
Hm, that's weird. I made it on purpose because we have an annotation "Non mutable static constants are not allowed in extern blocks". So I just made it mutable to remove this error message from the test. But now I see that Rust allows such constants, so it's just a wrong annotation that must be removed, I'll fix it in a separate PR.